src/components/StylePopup/StylePopup.js

```diff
@@ -13,15 +13,24 @@ import selectors from 'selectors';
 
 import './StylePopup.scss';
 
+const DATA_ELEMENTS = {
```

Comment:
Can we name this `dataElements` instead, to match the conventions we have in WebViewer?

Reply:
but these are just constants . shouldn't they be in upper case?

Reply:
Oh my bad, it should be `DataElements`. I'm not saying keep them all uppercase is wrong or something. Just want to make sure that for new constants they should match the naming convention we have in WebViewer.

Reply:
changed.
Seems I forgot that we agreed on a convention for constants a while.